examples/transfer-policy-check/Cargo.toml

```
@@ -0,0 +1,24 @@
+[package]
+name = "transfer-policy-check"
+version = "0.1.0"
+authors = ["Concordium <developers@concordium.com>"]
+edition = "2018"
+license = "MPL-2.0"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+# concordium-std = "*"
+concordium-std = {path = "../../concordium-std"}
+
+[features]
+default = ["std"]
+std = ["concordium-std/std"]
```

Either support `no_std`
```suggestion
[dependencies]
concordium-std = {path = "../../concordium-std", default-features = false}

[features]
default = ["std"]
std = ["concordium-std/std"]
```
or remove these features
```suggestion
[dependencies]
concordium-std = {path = "../../concordium-std"}
```